### PR TITLE
UICIRC-606: UI tests replacement with RTL/Jest for component LoanHist…

### PR DESCRIPTION
**Pull request by pavelspichonak**
## Purpose
Add RTL/Jest testing for `LoanHistoryForm` component in `src/settings/LoanHistory`

## Refs
https://issues.folio.org/browse/UICIRC-606

## Screenshots
Screen Shot 2022-01-17 at 3 50 53 PM

### CODE REVIEW

File: src/settings/LoanHistory/LoanHistoryForm.test.js [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+    it('should render closed loans label', () => {
+      expect(getById(testIds.closedLoansHeadline).getByText(labelIds.closedLoans)).toBeVisible();
+    });
+
+    it('should render anonymize label', () => {
+      expect(getById(testIds.closedLoans).getByText(labelIds.anonymize)).toBeVisible();
+    });
+
+    it('should render closed loans AnonymizingTypeSelectContainer component', () => {
+      expect(AnonymizingTypeSelectContainer).toHaveBeenNthCalledWith(1, expect.objectContaining({
```
Comment (Stanislau-Snapok) [inline src/settings/LoanHistory/LoanHistoryForm.test.js:135]: call order map for `AnonymizingTypeSelectContainer`

File: src/settings/LoanHistory/LoanHistoryForm.test.js [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+        name: 'treatEnabled',
+        type: 'checkbox',
+        id: 'treatEnabled-checkbox',
+        component: Checkbox,
+        label: labelIds.treat,
+      }, {});
+    });
+
+    it('should not render loans feefine section', () => {
+      expect(screen.queryByTestId('loansFeefineSection')).not.toBeInTheDocument();
```
Comment (Stanislau-Snapok) [inline src/settings/LoanHistory/LoanHistoryForm.test.js:153]: move `loansFeefineSection` into ids map

**Review by Stanislau-Snapok — APPROVED** [review]

File: test/jest/__mock__/stripesComponents.mock.js [hunk 3]
```diff
@@ -85,9 +85,17 @@ jest.mock('@folio/stripes/components', () => ({
     <div {...rest}>{children}</div>
   )),
   Modal: jest.fn(() => null),
-  Pane: jest.fn(({ children }) => (<div>{children}</div>)),
-  PaneFooter: jest.fn(({ ref, children, ...rest }) => (
-    <div ref={ref} {...rest}>{children}</div>
+  Pane: jest.fn(({ children, footer }) => (
+    <div>
```
Comment (pavelspichonak) [inline test/jest/__mock__/stripesComponents.mock.js:89]: extend Pane mock with all necessary props

File: src/settings/LoanHistory/LoanHistoryForm.test.js (hunk 1 above)
Comment (pavelspichonak) [reply src/settings/LoanHistory/LoanHistoryForm.test.js:135]: I'd like to discuss it. I specially did not create map

File: src/settings/LoanHistory/LoanHistoryForm.test.js (hunk 2 above)
Comment (pavelspichonak) [reply src/settings/LoanHistory/LoanHistoryForm.test.js]: Done. Thanks

File: test/jest/__mock__/stripesComponents.mock.js (hunk 3 above)
Comment (pavelspichonak) [reply test/jest/__mock__/stripesComponents.mock.js:89]: done

**Review by Dmitriy-Litvinenko — APPROVED** [review]

**Pull request merged**